doc/samples/samples_list.rst

```diff
@@ -27,6 +27,13 @@ The following table demonstrates the variants' configuration for the supported d
 
 `*` The `nRF54L15 PDK`_ does not support LoRa and FSK configurations.
 
+.. note::
+
+   Targets `nRF5340 DK`_  and `Thingy53`_  use `MCUBOOT_MODE_OVERWRITE_ONLY` configuration for bootloader. This means, that after the DFU image swap, there is no rollback mechanism in case new image is not functional.
+   This limitation is due to split between app and net core. If the default MCUBOOT mode to be set, the rollback would only apply to the app core, and not to the net core. This could lead to missmatch between app and net core.

```

Comment:
I would remove this line, the nrf53 limitations are known in the whole NCS, not Sidewalk specyfic. You can replace it with the link to generic NCS explonation (but I dont know where it is)